psydac/linalg/block.py

```diff
@@ -953,6 +953,8 @@ def copy(self, out=None):
                 Lij.copy(out=out[i,j])
 
         out.set_backend(self._backend)
+
+        return out
```

Comment:
Please add the `Returns` section to this method's docstring.

Reply:
5f47f54a59842a945a2c31611a6700e20e8aa920